todo-facebookoauth/FacebookOAuth/taco.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cordova-cli": "4.3.1"
```

Comment:
Cordova CLI version 5.0.1 is not found in the npm repository. 
Plugins with parameters (like the facebook plugin) were not working for Android using Cordova 5.1.1 on VS pre RTM versions. I just verified with latest VS and it is working, so I moved to cordova-cli 5.1.1